v2/tools/generator/internal/testcases/property_assignment_test_case.go

```diff
@@ -279,11 +280,14 @@ func (p *PropertyAssignmentTestCase) createTestMethod(
 		dst.NewIdent(errId),
 		astbuilder.CallQualifiedFunc("err", "Error"))
 
+	// match := cmp.Equal(subject, actual, cmpopts.EquateEmpty())
+	equateEmpty := astbuilder.CallQualifiedFunc(cmpoptsPackage, "EquateEmpty")
```

Comment:
minor: Maybe leave a comment referring to the open issue, or just explain the problem w.r.t. empty slices, so that viewers understand why this bit is key.